### ISSUE TITLE
Allow providing ATLANTIS_REPO_CONFIG_JSON as a YAML file

### ISSUE THREAD

**Issue by nitrocode** [terraform-aws-modules/terraform-aws-atlantis#204]
## Is your request related to a new offering from AWS?
No

## Is your request related to a problem? Please describe.
The problem is that people think the only way to provide this information is by using a json file which is untrue. Something like this can be used.

`server-atlantis.yaml`
```yaml
repos:
  - id: /.*/
    # allow repo level custom workflows
    allow_custom_workflows: true
    # allow repo level configs to override the following
    allowed_overrides:
      - apply_requirements
      - workflow
    apply_requirements:
      - approved
    workflow: default
```

`main.tf`
```hcl
module "atlantis" {
  source  = "terraform-aws-modules/atlantis/aws"
  version = "~> 2.30.0"

  custom_environment_variables = [
    # Override server config
    {
      "name" : "ATLANTIS_REPO_CONFIG_JSON",
      "value" : jsonencode(yamldecode(file("${path.module}/server-atlantis.yaml"))),
    },
  ]

  # ...
}
```

## Describe the solution you'd like.
We could provide a couple inputs

* `atlantis_repo_config_json`
* `atlantis_repo_config_yaml_file`

## Describe alternatives you've considered.
We could simply document the `jsonencode(yamldecode(file(` method

## Additional context
https://github.com/terraform-aws-modules/terraform-aws-atlantis/issues/116

**Comment by antonbabenko** [comment]
Agree, PR is welcome :)

**Comment by nitrocode** [comment]
Great! I just wanted to open discussion for the different methods to do this.

Documenting it would be easy but an input variable would probably be better. If no one has an opinion, then I'd like to put in a PR for an input `var.atlantis_repo_config_yaml_file`.

**Comment by antonbabenko** [comment]
I think it will be fine if we ask users to pass it like this:
```hcl
atlantis_repo_config_yaml_file  = jsonencode(yamldecode(file("${path.module}/server-atlantis.yaml")))
```

**Comment by antonbabenko** [comment]
Fixed in #205
